### ISSUE TITLE
Version skew with mp4v2 framework causes fail to build from source.

### ISSUE THREAD

**Issue by poetnerd** [sbooth/Max#5]
The mp4v2 SVN repository has been archived.  Apparently code.google.com is no longer a read/write community.  It is unclear whether mp4v2 is now orphaned, or if there is a new definitive upstream for it.

The current issue with Max is version skew:

The build instructions under sbooth/AudioFrameworks say to check out SVN revision 501.  This corresponds to mp4v2-2.0.  However the API used in Max seems to be 1.9.0, and the API is different.

As documented at:
http://mp4v2.googlecode.com/svn-history/r434/doc/1.9.0/api/group__mp4__file.html
1.9.0 defines the calls to have a verbosity flag that is no longer in the definitions found in file.h
from the 2.0.0 distribution.

The MP4_DETAILS bitmasks are also gone.
MP4_DETAILS_ALL
MP4_DETAILS_ERROR
MP4_DETAILS_WARNING
MP4_DETAILS_READ
MP4_DETAILS_WRITE
MP4_DETAILS_FIND
MP4_DETAILS_TABLE
MP4_DETAILS_SAMPLE
MP4_DETAILS_HINT
MP4_DETAILS_ISMA
MP4_DETAILS_EDIT

I presume that since the last distribution of Max was a 2009 build, that 1.9.0 was canon then and that's what Max was built with. Unfortunately, the mp4v2 project under AudioFrameworks won't build from 1.9.0 source.  There are missing source files, and we get a semantic error:

.../AudioFrameworks/mp4v2/mp4v2-src/src/bmff/typebmff.cpp:38:49: Cannot define or redeclare 'data' here because namespace 'bmff' does not enclose namespace 'Enum<mp4v2::impl::bmff::LanguageCode, 0>'

**Comment by poetnerd** [comment]
I hope I'm doing this right.
Assuming that the simplest, easiest thing is to utilize the Framework build already done with the AudioFrameworks tree, which uses 2.0.0, I just modified a few lines in a few files in Max to get it to comply with the slightly different API.  I've created a git gist file of the patch:

https://gist.github.com/poetnerd/eaf16a95650320187a1c

Hey Steve, this might close the issue.
